src/vm/codeman.cpp

```diff
@@ -5114,7 +5114,7 @@ BOOL NativeImageJitManager::JitCodeToMethodInfo(RangeSection * pRangeSection,
         }
 
         //Get the function entry that corresponds to the real method desc.
-        _ASSERTE(RelativePc >= RUNTIME_FUNCTION__BeginAddress(FunctionEntry));
+        _ASSERTE((RelativePc >= RUNTIME_FUNCTION__BeginAddress(FunctionEntry)));
```

Comment:
It is strange, could you please show me the error message you get when you don't include the additional parens? The `_ASSERTE` is defined as follows:
```C
  #define _ASSERTE(expr) _ASSERTE_MSG(expr, #expr)
```
and the `_ASSERTE_MSG` like this:
```C
  #define _ASSERTE_MSG(expr, msg)                                           \
        do {                                                                \
             if (!(expr)) {                                                 \
                PRE_ASSERTE                                                 \
                DbgAssertDialog(__FILE__, __LINE__, msg);                   \
                POST_ASSERTE                                                \
             }                                                              \
        } while (0)
```
Since the `_ASSERTE_MSG` already wraps the expression in parens when checking it and the expression doesn't contain commas that could confuse the expansion of the `_ASSERTE`, I don't understand why would it fail.